inc/spbc-settings.php

```diff
@@ -2302,23 +2302,23 @@ function spbc_field_scanner($params = array()){
 		'<div id="spbc_scaner_progress_overall" class="spbc_hide" style="padding-bottom: 10px; text-align: center;">'
 			.'<span class="spbc_overall_scan_status_get_hashes">'       .__('Receiving core hashes', 'security-malware-firewall')              .'</span> -> '
 			.'<span class="spbc_overall_scan_status_count_hashes_plug">'.__('Counting plugins and themes', 'security-malware-firewall')        .'</span> -> '
-			.'<span class="spbc_overall_scan_status_get_hashes_plug">'  .__('Receiving plugins and themes hashes', 'security-malware-firewall').'</span> -> '
+			.'<span class="spbc_overall_scan_status_get_hashes_plug">'  .__('Receiving plugin and theme hashes', 'security-malware-firewall').'</span> -> '
 			.'<span class="spbc_overall_scan_status_clear_table">'      .__('Preparing', 'security-malware-firewall')                          .'</span> -> '
 			.'<span class="spbc_overall_scan_status_count">'            .__('Counting files', 'security-malware-firewall')                     .'</span> -> '
 			.'<span class="spbc_overall_scan_status_scan">'             .__('Scanning for modifications', 'security-malware-firewall')         .'</span> -> '
-			.'<span class="spbc_overall_scan_status_get_hashes_approved">'  .__('Updating status for approved files', 'security-malware-firewall').'</span> -> ';
+			.'<span class="spbc_overall_scan_status_get_hashes_approved">'  .__('Updating statuses for the approved files', 'security-malware-firewall').'</span> -> ';
 
 			if($spbc->settings['scanner__signature_analysis'])
-				echo '<span class="spbc_overall_scan_status_count_modified_sign">'.__('Counting not checked', 'security-malware-firewall').'</span> -> '
+				echo '<span class="spbc_overall_scan_status_count_modified_sign">'.__('Counting files for signature analysis', 'security-malware-firewall').'</span> -> '
 				.'<span class="spbc_overall_scan_status_scan_modified_sign">'.__('Signature analysis', 'security-malware-firewall').'</span> -> ';
 			
 			if($spbc->settings['scanner__heuristic_analysis'])
-				echo '<span class="spbc_overall_scan_status_count_modified_heur">'.__('Count not checked', 'security-malware-firewall').'</span> -> '
+				echo '<span class="spbc_overall_scan_status_count_modified_heur">'.__('Counting files for heuristic analysis', 'security-malware-firewall').'</span> -> '
```

Comment:
This is not about heuristic. In this stage the plugin checks what files haven't been checked yet or been dropped to such state.

Reply:
Fixed.